script/utils/Script.sol

```diff
@@ -35,6 +35,10 @@ contract Script is BaseScript {
 }
 
 contract GovernanceScript is Script, GovernanceHelper {
+  function toRateFeedId(string memory rateFeedString) internal pure returns (address) {
+    return address(uint160(uint256(keccak256(abi.encodePacked(rateFeedString)))));
+  }
```

Comment:
nit: dupped from `script/utils/mento/Oracles.sol`

Reply:
duped because importing floating functions doesn't work in solidity v0.5 so they're not compatible. It will be nicer after I finish my OTHER BIG ONE to refactor the whole deployments repo 😂